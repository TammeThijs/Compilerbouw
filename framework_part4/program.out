__init:

testfunctie:
   floadc 0
   fstore 0
   floadc 1
   fstore 1
   floadc 2
   fstore 2

.const float  5.500000
.const float  1.100000
.const float  3.300000
