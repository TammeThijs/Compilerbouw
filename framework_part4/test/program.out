printInt:
main:
   esr 0
   iload_0
   iloadc_0
   ireturn

